Makefile

```diff
@@ -5,24 +5,21 @@ else
 	ENVBIN="bin"
 endif
 
-BASE := $(shell git merge-base HEAD origin/master)
-
 env:
-	virtualenv env
+	python -m virtualenv env
 
-requirements: env
```

Comment:
Why remove the virtual env here and in lint?

Reply:
this was to get github actions to work, which calls `make requirements`, `make lint`, and `make test`

github actions doesn't need a venv, though I probably could get it to work without making these changes.

also none of our other repos does this (I used omen2 as a template)